sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -540,6 +553,10 @@ private void runProxy(Boolean useTestContainers, int port, int grpcEventingBacke
         javaOptions.add("-Dkalix.proxy.eventing.grpc-backend.host=host.testcontainers.internal");
         javaOptions.add("-Dkalix.proxy.eventing.grpc-backend.port=" + grpcEventingBackendPort);
       }
+      if (settings.tracingEnabled){
```

Comment:
I thought that the idea to test tracing with ubiapp was to run the app with tracing enabled and try some e2e testing.

Reply:
I'm considering that as well. But since I saw the app test in CI with only `test` I thought on adding this.